wrangler-core/src/main/java/io/cdap/directives/column/Rename.java

```diff
@@ -82,4 +85,17 @@ public Mutation lineage() {
       .relation(source, target)
       .build();
   }
+
+  @Override
+  public Schema getOutputSchema(SchemaResolutionContext context) {
+    Schema inputSchema = context.getInputSchema();
+    return Schema.recordOf(
+      "outputSchema",
+      inputSchema.getFields().stream()
```

Comment:
nit: can we use `getField` function? https://github.com/cdapio/cdap/blob/develop/cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java#L783 rather than streaming through all fields?

Reply:
So the alternative approach is to create a new list of fields from inputSchema.getFields() and then iterate over every field to find the one to rename. We can't use any of the list compare/contains/replace methods since the Field object can't be compared directly (need to compare based on name manually)